versioninfo.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
The `versioninfo.json` name [seems to be hard-coded](https://github.com/josephspurrier/goversioninfo#usage) into `goversioninfo`, unfortunately.